src/devtoolsExtension.ts

```diff
@@ -28,25 +29,27 @@ export interface EnhancerOptions {
    */
   maxAge?: number
   /**
-   * - `undefined` - will use regular `JSON.stringify` to send data (it's the fast mode).
-   * - `false` - will handle also circular references.
-   * - `true` - will handle also date, regex, undefined, error objects, symbols, maps, sets and functions.
-   * - object, which contains `date`, `regex`, `undefined`, `error`, `symbol`, `map`, `set` and `function` keys.
-   *   For each of them you can indicate if to include (by setting as `true`).
-   *   For `function` key you can also specify a custom function which handles serialization.
-   *   See [`jsan`](https://github.com/kolodny/jsan) for more details.
+   * See detailed documentation at http://extension.remotedev.io/docs/API/Arguments.html#serialize
```

Comment:
The docs are so long that I felt like it wasn't worth copy/pasting the entire thing into this jsdoc and felt that it'd be more useful to just provide a link to the original docs

Reply:
We should do that the other way around. 
We can include Jsdoc comments directly into the documentation using "links" like `
[summary,remarks](docblock://mapBuilders.ts?token=ActionReducerMapBuilder.addMatcher)` in the documentation markdown files.
That way they will stay in sync - but they will also be fully available in autocomplete popups without an extra link.

Reply:
I'm confused. We do not own this documentation. The `serialize` property is just a passthrough to the Redux Devtools extension, so they own the docs, hence why I put a link to their docs here.

Reply:
Ah, good point. I had somehow assumed that we had our own docs on that like some other re-exports we do.

Reply:
Oh interesting. Yeah the docs here just point at the docs there. From https://redux-toolkit.js.org/api/configureStore#devtools it goes to https://github.com/zalmoxisus/redux-devtools-extension/blob/master/docs/API/Arguments.md#windowdevtoolsextensionconfig (which is the link I originally used in this comment but they have an official website, so you could also update this link in the docs).